lib/suspenders/generators/stylesheet_base_generator.rb

```diff
@@ -8,7 +8,7 @@ class StylesheetBaseGenerator < Rails::Generators::Base
 
     def add_stylesheet_gems
       gem "bourbon", "~> 5.0.0.beta.7"
-      gem "neat", "~> 2.0.0.beta.1"
+      gem "neat", "~> 2.0.0"
```

Comment:
Should we remove the patch number here, so we that we always get minor release updates? Specifying the patch with a twiddle-wakka equates to `['>= 2.0.0', '< 2.1.0']` (i.e. patch releases only, but not `2.1` for example)? By dropping the patch, it will equate to `['>= 2.0', '< 3.0.0']`.

See: http://guides.rubygems.org/patterns/#pessimistic-version-constraint